BXRetail-Terraform-pkg/README.md

```diff
@@ -104,6 +104,18 @@ If this is successful, you will see a new environment added to your PingOne orga
 
 Similarly, if you choose to deploy the app locally, the app may not load immediately depending on the amount of resources you allocated to the Docker runtime or the usual suspects; busy CPU, other resource intensive processes running, etc. 
 
+#### Deploying App Locally Using Existing PingOne Environment
+
+If you'd like to point this sample app to an existing PingOne environment and skip terraforming a new environment, you can do so by following the instructions below.
+
+Update the `deploy_app_to_k8s` variable in your `terraform.tfvars` file to "false".
+
+Navigate to the `.env.development` file in the `/bxretail-sample-app` directory, you'll need to uncomment the `REACT_APP_ENVID` and `REACT_APP_CLIENT` variables and set to your PingOne Environment Id and Application Client Id. 
```

Comment:
They also have to update REACT_APP_HOST to be https://localhost:5000. <-- this is used for redirect_uri, issuer, etc.

Reply:
@zhannaavanesova